scripts/generate_pip_deps_from_conda.py

```diff
@@ -32,6 +32,27 @@
     "seaborn-base": "seaborn",
     "sqlalchemy": "SQLAlchemy",
 }
+ADD_ENV_MARKERS = {
+    "pytables": " ; sys_platform != 'darwin' and platform_machine != 'arm64'",
```

Comment:
Maybe we can just add a comment here mentioning why this is done, something like:

```suggestion
    # pytables has no wheels available for macOS arm, and installing from source generally fails 
    # -> exclude from dev dependencies in this case
    "pytables": " ; sys_platform != 'darwin' and platform_machine != 'arm64'",
```